openpype/hosts/maya/plugins/publish/convert_gltf_shader.py

```diff
@@ -0,0 +1,91 @@
+import os
+from maya import cmds
+import pyblish.api
+
+from openpype.pipeline import publish
+
+
+class ConvertGLSLShader(publish.Extractor):
+    """
+    Converting StingrayPBS material to GLSL Shaders
+    specially for the glb export through Maya2GLTF plugin
+    """
+    order = pyblish.api.ExtractorOrder - 0.1
+    hosts = ["maya"]
+    label = "Convert StingrayPBS to GLTF"
+    families = ["gltf"]
+    optional = True
+
+    def process(self, instance):
+        meshes = cmds.ls(instance, type="mesh", long=True)
+        self.log.info("meshes: {}".format(meshes))
+        # load the glsl shader plugin
+        cmds.loadPlugin("glslShader", quiet=True)
+
+        for mesh in meshes:
+
+            # create glsl shader
+            glsl = cmds.createNode('GLSLShader')
+            glsl_shadingGrp = cmds.sets(name=glsl + "SG", empty=True,
+                                        renderable=True, noSurfaceShader=True)
+            cmds.connectAttr(glsl + ".outColor",
+                             glsl_shadingGrp + ".surfaceShader")
+
+            # load the maya2gltf shader
+            maya_publish = instance.context.data["project_settings"]["maya"]["publish"] # noqa
+
+            ogsfx_path = maya_publish["ConvertGLSLShader"]["ogsfx_path"]
+            if not ogsfx_path:
+                maya_dir = os.getenv("MAYA_APP_DIR")
+                if not maya_dir:
+                    raise RuntimeError("MAYA_APP_DIR not found")
+                ogsfx_path = maya_dir + "/maya2glTF/PBR/shaders/"
```

Comment:
How about not adding this extra logic but just setting the default settings path to `{MAYA_APP_DIR}/maya2glTF/PBR/shaders/`?